proto-google-cloud-bigtable-v2/src/main/java/com/google/bigtable/v2/BigtableProto.java

```diff
@@ -95,6 +95,42 @@ public static void registerAllExtensions(com.google.protobuf.ExtensionRegistry r
       internal_static_google_bigtable_v2_ReadModifyWriteRowResponse_descriptor;
   static final com.google.protobuf.GeneratedMessageV3.FieldAccessorTable
       internal_static_google_bigtable_v2_ReadModifyWriteRowResponse_fieldAccessorTable;
+  static final com.google.protobuf.Descriptors.Descriptor
```

Comment:
won't these (and other) generated changes get overwritten on the next generated PR?

Reply:
discussed with Igor offline, we will wait to merge this once the protos are updated in googleapis